python/hsfs/ge_validation_result.py

```diff
@@ -174,10 +185,38 @@ def expectation_config(self, expectation_config):
                 "Expectation config field must be stringified json or dict"
             )
 
-    def __str__(self):
+    @property
+    def validation_time(self) -> Optional[datetime.datetime]:
+        return self._validation_time
+
+    @validation_time.setter
+    def validation_time(self, validation_time: Optional[int]) -> None:
```

Comment:
Can use `util.convert_event_time_to_timestamp` here to make handling of time in the API more consistent.